extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/HttpAuthenticator.java

```diff
@@ -215,6 +217,30 @@ public Uni<SecurityIdentity> apply(HttpAuthenticationMechanism mech) {
                     });
         }
 
+        if (inclusiveAuth && strictInclusiveMode && pathSpecificMechanism == null) {
```

Comment:
I'm not seeing any lax-mode specific processing, what am I missing ?

Reply:
I have tests that prove it works. Lax mode is already in the main branch. No changes required.

Reply:
So additions are specific for the strict mode.